src/cli.js

```diff
@@ -17,7 +17,7 @@ const writeText = async (text, file) => await writeFile(file, text);
 const printText = text => console.log(text);
 const getPrettyText = (articles, queryString, options) => prettyArticles(articles, queryString, options);
 
-async function search (queryString, options) {
+export async function search (queryString, options) {
   const searcher = new Search();
 
   const articles = await getInput(options);
```

Comment:
GitHub isn't allowing me to comment on the collapsed parts of the code but the error would occur at line 74 within the getInput() function when executing JSON.parse(). It wasn't reading the combined JSON file with all the preprint data correctly.